docs/blocks/disks.mdx

```diff
@@ -0,0 +1,60 @@
+---
+name: Disks
+menu: Blocks
+
+---
+import { Playground, PropsTable } from 'docz'
+
+# Disks
+
+### Disk cards
+
+<Playground>
+  <div class="block-disks">
+    <div class="block-disks-item">
+      <div class="block-disks-item__lable">SSD_11</div>
+      <div class="block-disks-item__capacity">12 GB</div>
+    </div>
+    <div class="block-disks-item">
+      <div class="block-disks-item__lable">SSD_11</div>
+      <div class="block-disks-item__capacity">12 GB</div>
+    </div>
+    <div class="block-disks-item">
+      <div class="block-disks-item__lable">SSD_11</div>
+      <div class="block-disks-item__capacity">12 GB</div>
+    </div>
+    <div class="block-disks-item">
+      <div class="block-disks-item__lable">SSD_11</div>
+      <div class="block-disks-item__capacity">12 GB</div>
+    </div>
+    <div class="block-disks-item">
+      <div class="block-disks-item__lable">SSD_11yuuityutuytuytuytuytuytut-67866g</div>
+      <div class="block-disks-item__capacity">1212 GB</div>
+    </div>
+    <div class="block-disks-item">
+      <div class="block-disks-item__lable">SSD_11</div>
+      <div class="block-disks-item__capacity">12 GB</div>
+    </div>
+    <div class="block-disks-item">
+      <div class="block-disks-item__lable">SSD_11</div>
+      <div class="block-disks-item__capacity">12 GB</div>
+    </div>
+  </div>
+</Playground>
+
+##### Grouped disks
+
+<Playground>
+<div class="block-disks block-disks-grouped">
+  <div class="block-disks-item">
+    <div class="block-disks-item__lable">SSD_11</div>
+    <div class="block-disks-item__capacity">12 GB</div>
+  </div>
+  <div class="block-disks-item">
+    <div class="block-disks-item__lable">SSD_11</div>
+    <div class="block-disks-item__capacity">12 GB</div>
+  </div>
+</div>
+
+
+</Playground>
```

Comment:
`Group disks` can be replaced with `card-group` since these disks appear like `cards`. I am currently working on it. Will raise the PR soon.